agentcli/agentcli/cli.py

```
@@ -14,6 +14,10 @@
 import traceback
 import string
 
+# for python2 -> python3
+if sys.version_info[0] < 3:
+    input = raw_input
```

Ruff (0.15.4)

[error] Undefined name `raw_input`

(F821)